doc/howto/autoinstall-quickstart-s390x.rst

```diff
@@ -97,26 +101,29 @@ Add the default user to the ``kvm`` group:
 
     sudo usermod -a -G kvm ubuntu   # re-login to make the changes take effect
 
+Run the installation in a VM. Change ``<version-number>`` in the following command to match the release ISO you downloaded:
+
+.. code-block:: none
+
     kvm -no-reboot -name auto-inst-test -nographic -m 2048 \
         -drive file=disk-image.qcow2,format=qcow2,cache=none,if=virtio \
-        -cdrom ~/Downloads/ubuntu-22.10-live-server-s390x.iso \
+        -cdrom ~/Downloads/ubuntu-<version-number>-live-server-s390x.iso \
         -kernel ~/iso/boot/kernel.ubuntu \
         -initrd ~/iso/boot/initrd.ubuntu \
         -append 'autoinstall ds=nocloud-net;s=http://_gateway:3003/ console=ttysclp0'
 
-The above commands boot the virtual machine, download the configuration from the server
-(prepared in the previous step) and run the installation.
-
-The installer reboots at the end. The ``-no-reboot`` flag to ``kvm`` instructs ``kvm``
-to terminate on reboot. The procedure takes approximately 5 minutes.
+This command boots the VM, downloads the configuration from the server (prepared in the previous step) and runs the installation. The installer reboots at the end. The ``-no-reboot`` option to the ``kvm`` command instructs ``kvm`` to exit on reboot. This command takes approximately 5 minutes.
```

Comment:
I wonder if we should remove the reference to "takes approximately 5 minutes" outright.  Install times can vary wildly depending on hardware and network and what version is being installed.

Reply:
Ack